docs/modules/jenkins-shared-library/partials/odsComponentStageInfrastructure.adoc

```diff
@@ -0,0 +1,44 @@
+// Document generated by render-adoc.go from odsComponentStageInfrastructure.adoc.tmpl; DO NOT EDIT.
+
+odsComponentStageInfrastructure offers a complete pipeline for components created with the inf-terraform-aws or inf-terraform-azure quickstarter (https://github.com/opendevstack/ods-quickstarters/tree/master/inf-terraform-aws).
+It takes care of testing, deploying and reporting of infrastructure resources specified inside the component by running substages specified inside the quickstarters Makefile.
+
+List substages + explanation
+
+== Options
+
+[cols="1,2"]
+|===
+| Option | Description
+
+
+| *branch* +
+_String_
+|Branch to run stage for.
+ Example: `'master'`.
+ Next to exact matches, it also supports prefixes (e.g. `feature/`) and all branches (`*`).
+
+
+| *branches* +
+_List<String>_
+|Branches to run stage for.
```

Comment:
@nichtraunzer what exactly would the QS do, if a user names one or several branch or branches? Can you be a bit more concrete?

Reply:
that file is auto generated ... all the ods stages have that.

Reply:
I am thinking of creating more immediately understandable documentation for technical end-users. Might be good enough for now, some use case example wouldn't hurt. We can keep this as-is.